build/release/latest_release_branch.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+set -eEuo pipefail
+
+# Get list of release- branches
+RELEASE_BRANCHES="$(git branch -r | grep -v mergify | grep '/release-' | cut -d'/' -f2 | sort | uniq)"
+LATEST_RELEASE_BRANCH=""
+MAX_TRIES=5
+
+TRY=1
+until [ -n "${LATEST_RELEASE_BRANCH}" ] || [ ${TRY} -ge ${MAX_TRIES} ]; do
```

Comment:
```suggestion
until [ -n "${LATEST_RELEASE_BRANCH}" ] || [ ${TRY} -ne ${MAX_TRIES} ]; do
```
`TRY=1` and `MAX_TRIES=5`, `TRY` is never greater that `MAX_TRIES`? maybe we can use `-ne` or rever the vars

```suggestion
until [ -n "${LATEST_RELEASE_BRANCH}" ] || [ ${MAX_TRIES} -ge  ${TRY} ]; do
```

Reply:
`TRY` is incremented on every loop (see line 27) is it can get "greater equal" then `MAX_TRIES`.